docs/api-markdown-documenter/api-documentation-layout.js

```diff
@@ -18,25 +20,49 @@ import { AlertNode } from "./alert-node.js";
 const customExamplesSectionTitle = "Usage";
 const customThrowsSectionTitle = "Error Handling";
 
-const alphaWarning = SpanNode.createFromPlainText(
-	"WARNING: This API is provided as an alpha preview and may change without notice. Use at your own risk.",
-);
-const betaWarning = SpanNode.createFromPlainText(
-	"WARNING: This API is provided as a beta preview and may change without notice. Use at your own risk.",
-);
+// Temporary workaround for items tagged as `@alpha` (to mean "legacy").
+// This messaging should be changed back to standard "alpha" terminology once we have
+// cleaned up our tag meanings.
+function createAlphaWarning(apiItem) {
+	const packageName = apiItem.getAssociatedPackage().displayName;
+	return new AlertNode(
+		[
+			new SpanNode([
+				new PlainTextNode("To use, import via "),
+				CodeSpanNode.createFromPlainText(`${packageName}/legacy`),
+				new PlainTextNode("."),
+			]),
+		],
+		/* alertKind: */ "note",
+		/* title: */ "This API is provided as-is for existing users, but is not recommended for new users.",
+	);
+}
+
+function createBetaWarning(apiItem) {
+	const packageName = apiItem.getAssociatedPackage().displayName;
+	return new AlertNode(
+		[
+			new SpanNode([
+				new PlainTextNode("To use, import via "),
+				CodeSpanNode.createFromPlainText(`${packageName}/beta`),
+				new PlainTextNode("."),
+			]),
+		],
+		/* alertKind: */ "warning",
+		/* title: */ "This API is provided as a beta preview and may change without notice. Use at your own risk.",
```

Comment:
@kashms interested in your thoughts on this wording.

Reply:
Same as the last comment. I would remove the "Use at your own risk." part too :)